nx_neptune/instance_management.py

```diff
@@ -274,6 +284,64 @@ def create_na_instance(config: Optional[dict] = None):
         )
 
 
+
+def start_na_instance(graph_id: str):
+    """
+    Attempt to resume a remote Neptune Analytics instance with the provided graph_id,
+    on behalf of the configured IAM user.
+
+    Returns:
+        asyncio.Future: A Future that resolves with the graph_id,
+        represent the instance being deleted, or String literal Fail
+        in the case of exception.
+
+    Raises:
+        Exception: If the Neptune Analytics instance creation fails
+    """
+    # Instance deletion
+    na_client = boto3.client("neptune-graph")
+    response = na_client.start_graph(graphIdentifier=graph_id)
+
+    status_code = _get_status_code(response)
+    if status_code == 200:
+        fut = TaskFuture(graph_id, TaskType.START, _ASYNC_POLLING_INTERVAL)
+        asyncio.create_task(_wait_until_task_complete(na_client, fut), name=graph_id)
+        return asyncio.wrap_future(fut)
+    else:
+        fut = TaskFuture("-1", TaskType.NOOP, _ASYNC_POLLING_INTERVAL)
+        fut.set_exception(Exception(f"Invalid response status code: {status_code}"))
+        return asyncio.wrap_future(fut)
+
+
+
+def stop_na_instance(graph_id: str):
+    """
+    Attempt to stop a remote Neptune Analytics instance with the provided graph_id,
+    on behalf of the configured IAM user.
+
+    Returns:
+        asyncio.Future: A Future that resolves with the graph_id,
+        represent the instance being deleted, or String literal Fail
+        in the case of exception.
+
+    Raises:
+        Exception: If the Neptune Analytics instance creation fails
+    """
+    # Instance deletion
+    na_client = boto3.client("neptune-graph")
+    response = na_client.stop_graph(graphIdentifier=graph_id)
```

Comment:
should we validate the state of the graph before doing this?  or should we just assume that the user will check first, and that the stop_graph call will handle the state correctly.  

What happens if the graph is already stopped?  Should we report an error in that case or just return a success with no waiting required?